substratevm/mx.substratevm/mx_substratevm.py

```diff
@@ -418,7 +418,8 @@ def native_unittests_task():
     additional_build_args = [
         '-H:AdditionalSecurityProviders=com.oracle.svm.test.SecurityServiceTest$NoOpProvider',
         '-H:AdditionalSecurityServiceTypes=com.oracle.svm.test.SecurityServiceTest$JCACompliantNoOpService',
-        '-H:+AllowVMInspection'
+        '-H:+AllowVMInspection',
+        '--initialize-at-build-time=com.oracle.svm.test.jdk11.AbstractClassSerializationTest'
```

Comment:
I think that this is not a proper place to put a flag for a test. Take a look at this example https://github.com/oracle/graal/blob/master/substratevm/src/com.oracle.svm.test/src/META-INF/native-image/com.oracle.svm.test/native-image.properties#L1-L1